api/v1alpha1/oadp_types.go

```diff
@@ -115,13 +115,13 @@ type PodConfig struct {
 	// labels to add to pods
 	// +optional
 	Labels map[string]string `json:"labels,omitempty"`
-	// nodeSelector defines the nodeSelector to be supplied to podSpec
+	// nodeSelector defines the nodeSelector to be supplied to NodeAgent podSpec
```

Comment:
NIT: Remove `NodeAgent` from this kubebuilder comment/marker, `PodConfig` is not just specific for nodeAgent/Restic, it also used by Velero Pod.